packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -89,7 +91,8 @@ export const pollAppLogs = async ({
 
         // eslint-disable-next-line no-await-in-loop
         await useConcurrentOutputContext({outputPrefix: log.source, stripAnsi: false}, async () => {
-          if (log.event_type === 'function_run') {
+          // Use only log.log_type after https://github.com/Shopify/partners/pull/55178
+          if ((log.log_type === undefined ? log.event_type : log.log_type) === LOG_TYPE_FUNCTION_RUN) {
```

Comment:
That other PR will remove `event_type` from the API response, so this will mean that it will break the current behavior. I will merge the partners PR after this one, then open another CLI PR to only use `log_type`.

This way nothing *should break for anyone.